examples/v2/chat2.nim

```diff
@@ -303,6 +304,11 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
 
     await node.query(HistoryQuery(contentFilters: @[HistoryContentFilter(contentTopic: DefaultContentTopic)]), storeHandler)
```

Comment:
I think that accounts state will be affected when a history query is sent to another node (the account state of the querying node is changed on the queried node).  I suggest to keep track of accounts state and log them in the query proc as well. @oskarth wdyt?

Reply:
Ah right, this should actually probably use the `queryWithAccounting` function?

Reply:
Yea, I agree with @staheri14 on using the `queryWithAccounting` function, Since updates will be made to the accounting balance at intervals.

Reply:
`queryWithAccounting` is internally called by the `query` proc given that the `swap` protocol is enabled on the waku node https://github.com/status-im/nim-waku/blob/2691b3e50699ff3ab119a200b97cda01289b0146/waku/v2/node/wakunode2.nim#L341 
So I think no need to call `queryWithAccounting` directly, the `query` proc takes care of it.